model/Build/Properties/parameter.md

```diff
@@ -9,9 +9,11 @@ Property describing a parameter used in an instance of a build.
 ## Description
 
 parameter is a key-value of a build parameter and its value that
-was provided to the builder for a build instance. This is different from the
-[environment](environment.md) property in that the key and value are provided
-as command line arguments or a configuration file to the builder.
+was provided to the builder for a build instance.
+
+This is different from the [environment](environment.md) property in that
```

Comment:
I can remove this. The link has been there since the beginning https://github.com/spdx/spdx-3-model/blob/fdb9a143d9e2f1c90c76d02784dde69335859903/model/Build/Properties/parameters.md 

If needed, I can remove the link to parameter.md from environment.md as well.

Reply:
I will keep this for this PR. Another PR can be open two remove links in both parameter and environment, so they can look the same.